app/build.gradle

```diff
@@ -66,11 +47,15 @@ android {
         targetCompatibility 1.8
         sourceCompatibility 1.8
     }
+    lintOptions {
+        abortOnError false
```

Comment:
I don't _think_ this should be necessary. Were you having lint issues?

Reply:
still fails. either this or I guess I can try fixing the lints.

Reply:
Looks like it's related to the leaky EthereumJ dependency. We'll have to stick with the bandaid fix for now.